frontend/src/pages/projects/screens/detail/pipelines/PipelinesSection.tsx

```diff
@@ -13,29 +14,34 @@ const PipelinesSection: React.FC = () => {
     pipelinesServer: { initializing, installed, timedOut },
   } = usePipelinesAPI();
 
+  const [isPipelinesEmpty, setPipelinesIsEmpty] = React.useState(false);
```

Comment:
I missed this earlier, the names are different. Any specific reason why we can't use `setIsPipelinesEmpty`? The same applies to all the other places where this variable is being used.

Reply:
My bad, missed that :(